ambry-tools/src/main/java/com.github.ambry/store/ConsistencyCheckerTool.java

```diff
@@ -151,21 +155,23 @@ public static void main(String args[]) throws Exception {
       }
       Time time = SystemTime.getInstance();
       Throttler throttler = new Throttler(config.indexEntriesToProcessPerSec, 1000, true, time);
+      StoreKeyConverterFactory storeKeyConverterFactory = Utils.getObj(serverConfig.serverStoreKeyConverterFactory, properties, null);
```

Comment:
pass the metric registry? `clusterMap.getMetricRegistry()`

Reply:
StoreKeyConverterFactory doesn't currently use it, but another implementation may.  Fixed.